.github/terraform/main.tf

```
@@ -0,0 +1,313 @@
+  network_interface_ids = [azurerm_network_interface.nic[count.index].id]
+  size                  = var.azure_instance_type
+  admin_username        = var.azure_ssh_user
+
+  os_disk {
+    name                 = "OsDisk_${count.index}"
+    caching              = "ReadWrite"
+    storage_account_type = "Standard_LRS"
+  }
+
+  admin_ssh_key {
+    username   = var.azure_ssh_user
+    public_key = file("${var.local_key_path}/${var.azure_key_name}.pub")
+  }
+
+
+  source_image_reference {
+    publisher = "Canonical"
+    offer     = "UbuntuServer"
+    sku       = "16.04-LTS"
+    version   = "latest"
+  }
+
+
+  identity {
+    type         = "UserAssigned"
+    identity_ids = [azurerm_user_assigned_identity.hazelcast_reader.id]
+  }
+
+
+  connection {
+    host        = azurerm_public_ip.publicip[count.index].ip_address
+    user        = var.azure_ssh_user
+    type        = "ssh"
+    private_key = file("${var.local_key_path}/${var.azure_key_name}")
+    timeout     = "120"
+    agent       = false
+  }
+
+  provisioner "remote-exec" {
+    inline = [
+      "mkdir -p /home/${var.azure_ssh_user}/jars",
+      "mkdir -p /home/${var.azure_ssh_user}/logs",
+      "while [ ! -f /var/lib/cloud/instance/boot-finished ]; do echo 'Waiting for cloud-init...'; sleep 1; done",
+      "sudo apt-get update",
+      "sudo apt-get -y install openjdk-8-jdk wget",
+    ]
+  }
+
+  provisioner "file" {
+    source      = "scripts/start_azure_hazelcast_member.sh"
+    destination = "/home/${var.azure_ssh_user}/start_azure_hazelcast_member.sh"
+  }
+
+  provisioner "file" {
+    source      = "scripts/verify_member_count.sh"
+    destination = "/home/${var.azure_ssh_user}/verify_member_count.sh"
+  }
+
+  provisioner "file" {
+    source      = "~/lib/hazelcast-azure.jar"
+    destination = "/home/${var.azure_ssh_user}/jars/hazelcast-azure.jar"
+  }
+
+  provisioner "file" {
+    source      = "~/lib/hazelcast.jar"
+    destination = "/home/${var.azure_ssh_user}/jars/hazelcast.jar"
+  }
+
+  provisioner "file" {
+    source      = "hazelcast.yaml"
+    destination = "/home/${var.azure_ssh_user}/hazelcast.yaml"
+  }
+
+  provisioner "remote-exec" {
+    inline = [
+      "cd /home/${var.azure_ssh_user}",
+      "chmod 0755 start_azure_hazelcast_member.sh",
+      "./start_azure_hazelcast_member.sh  ${var.azure_tag_key} ${var.azure_tag_value} ",
+      "sleep 5",
+    ]
+  }
+}
+
+resource "null_resource" "verify_members" {
+  count = var.member_count
+  depends_on = [ azurerm_linux_virtual_machine.hazelcast_member ]
+
+  connection {
+    type = "ssh"
+    user = var.azure_ssh_user
+    host = azurerm_linux_virtual_machine.hazelcast_member[count.index].public_ip_address
+    timeout = "180s"
+    agent = false
+    private_key = file("${var.local_key_path}/${var.azure_key_name}")
+  }
+
+  provisioner "remote-exec" {
+    inline = [
+      "cd /home/${var.azure_ssh_user}",
+      "tail -n 20 ./logs/hazelcast.stdout.log",
```

Regarding the change in start_member.sh:
```suggestion
      "tail -n 20 ./logs/hazelcast.logs",
```